DeviceManager/ImportHandler.py

```diff
@@ -26,9 +26,26 @@
 
 class ImportHandler:
 
+    kafka_handler = None
+
     def __init__(self):
         pass
 
+    @classmethod
+    def verifyInstance(cls, kafka):
```

Comment:
The verifyInstance method repeats in several places. Maybe it could be in one place where the handles extend a class. I do not know well.

Reply:
I don't know much about python.
But I had thought more of a parent class (for example Handler.py) where all the other somethingHandler could extend them.
And in it would remain in an attribute the instance, without the need to pass the cls, something like a singleton. But as I said, I don't know much about python and your solution seems ok to me.